executor/infoschema_reader.go

```diff
@@ -2011,6 +2013,14 @@ func (e *memtableRetriever) setDataForClientErrorsSummary(ctx sessionctx.Context
 	return nil
 }
 
+func (e *memtableRetriever) setDataForStmtSummaryEvicted(ctx sessionctx.Context, tableName string) error {
```

Comment:
Look like this function is useless? just use `e.rows = stmtsummary.StmtSummaryByDigestMap.ToEvictedCountDatum()` in line#153?

Reply:
Good idea, I wasn't aware of it.